docs/src/main/sphinx/release/release-475.md

```diff
@@ -0,0 +1,61 @@
+# Release 475 (16 Apr 2025)
+
+## General
+
+* Publish additional metrics for input tables in event listener. ({issue}`25475`)
+* Expose dynamic filter statistics in the `QueryCompletedEvent`. ({issue}`25575`)
+* Improve scalability of inline data encoding in the parallel client protocol. ({issue}`25439`)
+* Improve performance of queries involving the `exclude_columns` table function. ({issue}`25117`)
+* Disallow dropping they `system` catalog. ({issue}`24745`)
+* Fix occasional query failures when adaptive planning is enabled. ({issue}`25411`)
+* Fix incorrect results when using window functions with `DISTINCT`. ({issue}`25434`)
+* Fix query failures with `EXCEEDED_LOCAL_MEMORY_LIMIT` errors due to incorrect memory accounting. ({issue}`25600`)
+* Properly handle inline session properties for `EXPLAIN` queries. ({issue}`25496`)
+
+## Security
+
+* Fix impersonation access control when access is granted via the role. ({issue}`25166`)
+
+## BigQuery connector
+
+* Add support limiting the max parallelism via the `bigquery.max-parallelism` config property. ({issue}`25422`)
+* Fix queries getting stuck when reading large tables. ({issue}`25423`)
+
+## Delta Lake connector
+
+* Improve performance when filtering on `$path`, `$file_modified_time` and `$file_size` columns. ({issue}`25369`)
+* Improve performance of scans on delta lake tables with v2 checkpoints. ({issue}`25469`)
+
+## Hive connector
+
+* Add support for showing column comments on Hive views. ({issue}`23845`)
+* Support multiple predicate values for injected partition columns on tables using partition projection. ({issue}`17641`)
+* Fix potential failures or incorrect results when querying partitioned tables using the OpenX JSON SerDe. ({issue}`25444`)
+
+## Iceberg connector
+
+* Add `system.iceberg_tables` system table to allow listing only Iceberg tables. ({issue}`25136`)
+* Add support for [deletion vectors](https://iceberg.apache.org/spec/#deletion-vectors). ({issue}`24457`)
+* Add support for IAM role authentication REST catalog. ({issue}`25002`)
+* Add support for lock-free commit in Hive catalog. ({issue}`25445`)
```

Comment:
@ebyhr, can you rephrase in terms of user-visible impact? If this is a performance improvement, something like "Improve performance when xxx"

Reply:
Wouldn't one query fail anyway? Or are we reconciling the changes and trying again when that happens?